sections/main-search.liquid

```diff
@@ -256,7 +256,7 @@
                                 </h3>
                               </div>
                               <div class="card__badge {{ settings.badge_position }}">
-                                <span class="badge color-background-1">{{ 'templates.search.page' | t }}</span>
+                                <span class="badge color-{{ settings.color_schemes | first }}">{{ 'templates.search.page' | t }}</span>
```

Comment:
New pattern introduced with this. In a few files already, this would be the type of thing, if we're keeping it that we will need to share with all devs/designers. 

I'm not sure we need the extra caution as I don't feel like it's very likely that we re order our color schemes. 

So I'm torn between the extra safety it adds vs the new pattern.

Reply:
I like the safety it gives if a merchants ever rename the color ids to match their reality or if they remove it, swap it in the code. For the rest I'll leave it to your expertise! 

It's less likely a pattern we encourage with the new system, to reorder the schemes, mostly that the first `Scheme 1` became the new baseline starting point for the whole site and this was our only way of doing it for MVP until we learn more how merchants will use, etc. 

In our system, `Scheme 1` is the parent of all the schemes and the subsequent one are meant to create diversity and rhythm where needed.